tests/php/test-class-amp-core-block-handler.php

```diff
@@ -177,4 +177,128 @@ public function test_ampify_cover_block() {
 		$this->assertStringNotContains( 'width=', $content );
 		$this->assertStringNotContains( 'height=', $content );
 	}
+
+	/**
+	 * Test process_categories_widgets.
+	 *
+	 * @covers AMP_Core_Block_Handler::process_categories_widgets()
+	 * @see WP_Widget_Categories
+	 */
+	public function test_process_categories_widgets() {
+		$instance_count = 2;
+
+		ob_start();
+		for ( $i = 0; $i < $instance_count; $i++ ) {
+			the_widget(
+				'WP_Widget_Categories',
+				[ 'dropdown' => '1' ],
+				[]
+			);
+		}
+		$html = ob_get_clean();
+
+		$dom = AMP_DOM_Utils::get_dom_from_content( $html );
+
+		/**
+		 * Elements.
+		 *
+		 * @var DOMElement $select
+		 * @var DOMElement $form
```

Comment:
```suggestion
		 * @var DOMElement $selects
		 * @var DOMElement $forms
```

Reply:
This is intended actually. The type defs are for when looping over these below.